cmd/convert.go

```diff
@@ -97,7 +145,7 @@ func init() {
 	convertCmd.Flags().MarkHidden("deployment")
 
 	// OpenShift only
-	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", true, "Generate an OpenShift deploymentconfig object")
+	convertCmd.Flags().BoolVar(&ConvertDeploymentConfig, "deployment-config", false, "Generate an OpenShift deploymentconfig object")
```

Comment:
defaulting controllers is taken care in `validateControllers` https://github.com/kubernetes-incubator/kompose/blob/master/pkg/app/app.go#L136